### ISSUE TITLE
sass error when running gulp styles

### ISSUE THREAD

**Issue by haelbichalex** [SnowdogApps/magento2-frontools#301]
When I'm running the `gulp styles` command I get the following error:

```
 Error: ../../../var/view_preprocessed/frontools/frontend/Snowdog/all-possible-options/styles/vendor/magento-ui/_utilities.scss
Error: You may not @extend an outer selector from within @media.
       You may only @extend selectors within the same directive.
       From "@extend .abs-add-clearfix-desktop" on line 254 of ../../../var/view_preprocessed/frontools/frontend/Snowdog/all-possible-options/Magento_Sales/styles/_module.scss
        on line 66 of ../../../var/view_preprocessed/frontools/frontend/Snowdog/all-possible-options/styles/vendor/magento-ui/_utilities.scss
>>     &:before,

   ----^
```

**Comment by will-b** [comment]
@haelbichalex experiencing the same, think this is due to an update to a dependency.
Installing via yarn instead of npm seems to be workaround.

**Comment by Igloczek** [comment]
This issue is not related to Frontools.

Hotfix for SASS Blank theme released as version 1.0.5

You can check details here - https://github.com/SnowdogApps/magento2-theme-blank-sass/pull/185
